src/test/java/uk/gov/companieshouse/web/accounts/service/transaction/impl/TransactionServiceImplTests.java

```diff
@@ -1,5 +1,5 @@
 package uk.gov.companieshouse.web.accounts.service.transaction.impl;
-
+/*
```

Comment:
These tests need to be uncommented and updated inline with new functionality

Reply:
Implemented all suggested changes. Will still need to confirm with someone about the wording of the validation message (but have updated it to begin with the words "Tell us").